### ISSUE TITLE
Option to disable moving windows to corners

### ISSUE THREAD

**Issue by YANOUSHek** [nikitabobko/AeroSpace#462]
This might be a really weird question but I'll post it anyway. Currently when using multiple workspaces the windows that are on the same monitor but not in the active workspace are moved off to one of the bottom corners of the monitor. Currently this poses a problem for me and I can't use multiple workspaces on my main monitor because I don't have a "good corner".

I was thinking: since the idea of a Tiling Window Manager is to maximize the space used by apps, it always covers the whole screen with windows. If you're not using gaps between windows (like I do) it kind of doesn't matter if the other windows are hidden in the corners or just left underneath the active ones – they won't be visible. Would that be an option that could be easily added to Aerospace? To just leave the inactive windows on the monitor and only "bring forward" the ones that are in the active workspace?

**Comment by nikitabobko** [comment]
**(1)** How are going to focus empty workspace?

> it kind of doesn't matter if the other windows are hidden in the corners or just left underneath the active ones – they won't be visible

**(2)** It's not true because macOS windows "overlap stack" is not aligned with AeroSpace representation. Consider the following case:
```
workspace 1 (focused)
  h_tiles
    window 1 (focused)
    window 2
workspace 2 ("in background")
  h_tiles
    window 3
```

And macOS windows "overlap stack" (the windows on top are the most "recent"):
```
window 1
window 3
window 2
```

AeroSpace could try to "raise"/"focus" all windows of the workspace when it focuses the workspace, but in that case the focus will flicker.

Besides, focusing and "raising" windows is probably the most invasive and the buggiest API that I had to deal with (e.g. #101).

In the initial versions of AeroSpace, I played with the exact idea you propose, and I couldn't make it work. I tried to call "focus window" API several times in a row for different windows (to "raise" windows of the workspace), and very often a wrong windows ends up being focused when you switch workspaces. Though maybe I didn't try hard enough.

Related issue: #4
